apps/service-portal/src/components/ApplicationErrorBoundary/ApplicationErrorBoundary.tsx

```diff
@@ -27,7 +27,7 @@ export class ApplicationErrorBoundary extends PureComponent<
   }
 
   componentDidCatch(error: Error) {
-    if (window.location.origin === 'http://localhost:4200') {
+    if (window.location.origin.includes('http://localhost')) {
```

Comment:
We should probably also [disable Sentry](https://github.com/island-is/island.is/blob/main/apps/application-system/form/src/main.tsx#L9) if not running in a production environment